src/MOIWrapper.jl

```diff
@@ -374,3 +374,40 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+const statmap = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
+                     [MOI.BASIC, MOI.BASIC, MOI.NONBASIC_AT_UPPER, MOI.NONBASIC_AT_LOWER,MOI.SUPER_BASIC, MOI.NONBASIC]))
+function MOI.get(instance::Optimizer, ::MOI.ConstraintBasisStatus, i::LQOI.LCI{T}) where T <: Union{LQOI.LE, LQOI.GE, LQOI.EQ}
+    row = instance[i]
+    val = get_row_status(instance.inner, row)
+    stat = statmap[val]
+    # Single sided constraints, should not specify lower or upper
+    if stat == MOI.NONBASIC_AT_LOWER || stat == MOI.NONBASIC_AT_UPPER
+        return MOI.NONBASIC
+    end
+    return stat
+end
+
+function MOI.get(instance::Optimizer, ::MOI.ConstraintBasisStatus, ::LQOI.SVCI{T}) where T <: LQOI.EQ
+    return MOI.NONBASIC
+end
+
+function MOI.get(instance::Optimizer, ::MOI.ConstraintBasisStatus, ci::LQOI.SVCI{T}) where T <: LQOI.IV
+    col = instance.variable_mapping[instance[ci]]
+    return statmap[get_column_status(instance.inner, col)]
+end
+
+#Single sided constraints returns NONBASIC instead of NONBASIC_AT_X
+const statmap_LE = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
```

Comment:
Is it possible to avoid repeating this mapping in three places? E.g., using `if` statements to filter out the `NONBASIC_AT_X` as is done above.